src/cli.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import click
+from pathlib import Path
+from src.comparsion import compare_and_output
+
+def validate_paths(input_path: str, output_path: str) -> None:
+    input_path = Path(input_path)
+    
+    if not input_path.is_dir():
```

Comment:
If the directory exists, but the files we are analyzing are not there, we'll get an error:
`FileNotFoundError: [Errno 2] No such file or directory: 'input\\clojure_data\\atomsClojure.csv'`

So, we should also check if the exist and exit if that is not the case

Reply:
For clojure data, yes it is reasonable to check at the early stage so that user will not wait until the error shows up.

However, for the other two, the check seems unnecessary. The change I propose is to check whether all three folders at least contains something, will that be helpful?

Reply:
My advice would be to test all cases. We don't want a nice error to be raised if one or more of these files is missing. Actually, having only two should be sufficient. However, in that case, we want to skip including the missing one in our comparison

Reply:
Im sorry that I still didn't fully understand, 

Could you specify having which two would be sufficient? For instance for postIncr comparison, checking for both codeQL and coccinelle would be sufficient as clojure is just one file (all or nothing to compare). The same applies to all other 10 comparisons. ?

Or for each comparison category, either codeQL&clojure or clojure&coccinelle is sufficient? In that case I will use a hashmap or something to keep track of the # of count for each category. If the count is greater than 1 then there will be an output file for that, otherwise no.

Update on the progress: all other feedbacks have been addressed.

Reply:
So what I'm thinking is that if you should be able to compare any of them with whichever one you want. So, clojure and CodeQL, clojure and Coccinelle, CodeQL and Coccinelle. So if one set of files is missing, that shouldn't be an error. If there is only one, or none, print a nice error and quit